pkg/debuggers/remote/utils.go

```diff
@@ -37,13 +37,21 @@ func GetPort(pid int) (int, error) {
 	return port, nil
 }
 
-func GetPortOfJavaProcess(pid int) (int, error) {
+func GetPortOfJavaProcess(pid int, env map[string]string) (int, error) {
 
 	args, err := utils.GetCmdArgsByPid(pid)
 	if err != nil {
 		log.WithFields(log.Fields{"pid": pid, "err": err}).Error("Can't get command line arguments")
 		return 0, err
 	}
+	s, ok := env["JAVA_TOOL_OPTIONS"]
```

Comment:
@rverma-jm do you anticipate other environment variables being needed? I think it would be sufficient to check for this environment variable on the target pod spec instead of passing it through the interface

Reply:
Yes. I wanted to move this logic to individual debuggers but with current structure it become a refactoring task

Reply:
are we good with conversation, can we mark it resolved? Any todo here?